config/rails.yml

```diff
@@ -1,5 +1,9 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.5
```

Comment:
rubocop-rails_config supports Ruby 2.3 or higher.
https://github.com/toshimaru/rubocop-rails_config/blob/v0.5.1/rubocop-rails_config.gemspec#L13

I think it will be 2.3 if `TargetRubyVersion` will be updated.

Cc @toshimaru

Reply:
OK - I've changed it to 2.3. That was just a straight copy from the Rails config though, so the version will be different.